lib/Benchmark/Runner.php

```diff
@@ -82,6 +86,33 @@ public function run(RunnerContext $context)
         $suiteEl->setAttribute('date', date('c'));
         $suiteEl->setAttribute('config-path', $this->configPath);
         $suiteEl->setAttribute('retry-threshold', $context->getRetryThreshold($this->retryThreshold));
+        $envEl = $suiteEl->appendElement('env');
+        $unameEl = $envEl->appendElement('unname');
+        $vcsInfo = $this->vcsDetector->getVcsInformation();
+        foreach (array(
+            'os' => 's',
+            'host' => 'n',
+            'release' => 'r',
+            'version' => 'v',
+            'machine' => 'm',
+        ) as $key => $mode) {
+            $unameEl->setAttribute($key, php_uname($mode));
+        }
+        $vcsEl = $envEl->appendElement('vcs');
+        $vcsEl->setAttribute('system', $vcsInfo->getSystem());
+        $vcsEl->setAttribute('branch', $vcsInfo->getBranch());
+        $vcsEl->setAttribute('version', $vcsInfo->getVersion());
+        $phpEl = $envEl->appendElement('php');
+        $phpEl->setAttribute('version', PHP_VERSION);
+
+        if (!stristr(PHP_OS, 'win')) {
+            $load = sys_getloadavg();
```

Comment:
Note this information is pretty useless without knowing the number of CPUs the sytstem has.